perfkitbenchmarker/providers/aws/aws_dynamodb.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2018 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module containing class for AWS' dynamodb tables.
```

Comment:
Would you create a 'NoSQL: Key-value' base resource so that we can specify both GCP(Cloud Datastore, Cloud Bigtable) and AWS(	DynamoDB) in the same manner?

Reply:
do you have a flagged command line example of how this would be used?

Reply:
Like https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/1505
file: perfkitbenchmarker/managed_relational_db.py
Maybe create one like this perfkitbenchmarker/key_value_db.py

Reply:
I'm not familiar enough with the other key-value databases to know what they have in common, that we could use in such a file.

Reply:
That seems fair. We can add a common base class when another implementation is added.